control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -77,6 +78,8 @@ type Reconciler struct {
 	// NewKafkaClusterAdminClient creates new sarama ClusterAdmin. It's convenient to add this as Reconciler field so that we can
 	// mock the function used during the reconciliation loop.
 	NewKafkaClusterAdminClient kafka.NewClusterAdminClientFunc
+
+	KafkaFeatureFlags *config.KafkaFeatureFlags
```

Comment:
I don't see it actually used, are you planning to do that in a follow up PR?

Reply:
Yes, it will come in another PR https://github.com/knative-sandbox/eventing-kafka-broker/pull/2615